azerothcore/azerothcore-wotlk
# fix(Core/Quest): Fix NPC respawn for quest Tome of Valor

## Changes Proposed:
-  Set instant despawn & respawn, making the quest abled to be turned in without waiting 5minutes

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8323
- Closes https://github.com/chromiecraft/chromiecraft/issues/1978

## SOURCE:

https://youtu.be/xsehGRGIlx0?t=209
https://youtu.be/02oHynd015c?t=180
https://youtu.be/73F_9Q28gk4?t=246

## Tests Performed:

- Tested ingame
-  Build Passes

> ========== Build: 2 succeeded, 0 failed, 16 up-to-date, 1 skipped ==========


## How to Test the Changes:


1. Be an alliance paladin min level 20
2.` .quest add 1650`
3.` .go c 66979`
4. Turn in and accept the chained quest.
5. Defend her however you wish and complete the quest
6. Follow her, notice she instantly respawns. _(issue was taking 1 min despawn, 5min respawn)_

## Known Issues and TODO List:


- Port to SAI to finalize cosmetics, such as continued walking after turned in quest and assistance in fight


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
